tests/cypress/integration/actions_tasks3/case_44_changing_default_value_for_attribute.js

```diff
@@ -46,8 +46,12 @@ context('Changing a default value for an attribute.', () => {
                 cy.contains(new RegExp(`^${additionalLabel}$`))
                     .parents('.cvat-constructor-viewer-item')
                     .find('[aria-label="edit"]')
-                    .click({ force: true });
+                    .should('be.visible')
+                    .then((e) => {
+                        cy.get(e).click();
+                    });
```

Comment:
@dvkruchinin 
Could you please explain how does it fix the issue?

Reply:
I think that the issue was due to the fact that the click on the ```[aria-label="edit"]``` was carried out with the argument ```force: true``` and at this time attributes ```cvat-attribute-id``` haven't received positive values yet. When I cleaned up ```force: true``` then I got an error ```Detached from the DOM``` when click on ```[aria-label="edit"]```. Perhaps this is one of the ```react``` reactions. I tested this in my fork for about 20 restarts. Unfortunately, there is no complete certainty that this will completely solve the problem. For a similar issue, the issue is still open in the cypress repository.

Reply:
Okay, lets look how it will behave in real cases